sdk/simulator-service/src/strategies/refinanceLendingToLending/RefinanceLendingToLendingAnyPair.ts

```diff
@@ -112,7 +113,7 @@ export async function refinanceLendingToLending(
         depositAmount: isCollateralSwapSkipped
           ? position.collateralAmount
           : ctx.getReference(['SwapCollateralFromSourcePosition', 'received']),
-        borrowAmount: isDebtSwapSkipped
+        borrowAmount: isDebtTokenSame
```

Comment:
not sure about L146 - should it be`isDebtTokenSame` ?

Reply:
The swap from target debt to flashloan amount should only be done if the target debt token is different from the source debt token and the amount is not 0, which is exactly ` isDebtSwapSkipped` value